### DISCUSSION TITLE
Agentless Client

### DISCUSSION THREAD

**Discussion by CountryByte** [Ylianst/MeshCentral#6606] [Ideas]
I was hoping there is enough need for agentless Clients.  I would love to be able to manually add devices with web pages, telnet, ssh, embedded VNC, ILO Cards, etc..   Where I could manually enter in the info but be able to access these devices and grant access to others as needed.  WHer it just pings regularly that the device is alive.

**Comment by si458** [comment]
Meshcentral doesn't quite do everything u think it does

U create 4 different types of meshgroups for ur devices

Meshagent groups for computers u install the meshagent on and also physical intel amt devices

Intel amt only groups for computers that support intel amt and u want physical control over those devices, including bios but can't install the meshagent on

Local groups which is a group where u enter the remote devices dns or ip address and it allows u to ssh/rdp/vnc only to those machines (no telnet/ilo etc)

Relay groups which is a group of local devices that Meshcentral can access by using a device which is from a meshagent group and has the meshagent installed


Edit. Meshcenteal also doesn't ping local device groups to see if they are online or not...
